src/content/site.css

```diff
@@ -0,0 +1,3 @@
+body {
+  font-family: monospace;
+  font-size: 2rem; }
```

Comment:
Is this css autogenerated from the src/sass/site/site.scsss? Should it be not included in the repo?